gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionSha256Sum=8de6efc274ab52332a9c820366dd5cf5fc9d35ec7078fd70c8ec6913431ee610
```

Comment:
We need to add respective checksum to let the Gradle be cached on CI: https://gradle.org/release-checksums/

Reply:
Strange; I added it when I updated the version; I guess `./gradlew wrapper` removed it 😦

Reply:
Correct. It does remove it. We have to add it manually afterwards.
Probably question to Gradle team 😄